### PR TITLE
feat: Pagination and redirection to `Explore`  in mobile view

### LINKED ISSUE #3122: Improvements in Browse Events Section
**Describe the bug**

- [x] Mobile View does not redirect to the browse events section.
- [x] No pagination in the section.
- [x] Hide `Filters` for the mobile view as done in `Event Brite`.
- [ ] Add an Option to `Hide/Unhide` Map for Small Screens

### LINKED ISSUE #3139: Pagination and Right Link Redirection in mobile device for Explore Events
Sub Issue of #3122

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #3139 

#### Short description of what this resolves:

- Implemented Pagination in `Browse Events`.
- Redirection to `Explore` in mobile devices

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by kushthedude** [comment]
@CosmicCoder96 @niranjan94 Review Please

**Comment by kushthedude** [comment]
@CosmicCoder96 @niranjan94 @pradeepgangwar Review Please

File: app/routes/explore.js [hunk 1]
```diff
@@ -145,11 +145,17 @@ export default Route.extend({
...
-      include : 'event-topic,event-sub-topic,event-type'
+    return this.infinity.model('event', {
+      include      : 'event-topic,event-sub-topic,event-type',
+      filter       : filterOptions,
+      sort         : 'starts-at',
+      perPage      : 6,
+      startingPage : 1,
+      perPageParam : 'page[size]',
+      pageParam    : 'page[number]',
+      store        : this.store
```
Comment (CosmicCoder96) [inline app/routes/explore.js]: this is not required.

Comment (kushthedude) [reply app/routes/explore.js]: @CosmicCoder96 did the changes, Please review!

**Comment by mariobehling** [comment]
Please add a screenshot in all PRs!

**Comment by kushthedude** [comment]
@mariobehling Since the following is infinity loader I think the following ss will work
image

**Pull request merged**